src/main/java/com/example/pfairplayservice/jpa/model/MemberEntity.java

```diff
@@ -29,30 +31,35 @@
     private String uid;
 
     @Column(nullable = false)
+    @Length(min = 6, max = 10)
     private String id;
 
     @Column(nullable = false)
+    @Length(min = 6, max = 10)
     private String password;
 
     @Column(nullable = false)
+    @Length(min = 2, max = 10)
     private String name;
 
-    @Column(nullable = false)
-    private Date birthday;
+    @Column(nullable = false, length = 8)
+    private Integer birthday;
```

Comment:
Why integer?  Do you mean timestamp format?
If then, there is [Timestamp](https://docs.oracle.com/javase/8/docs/api/java/sql/Timestamp.html) type.

It is a good reference to express time in JPA.
https://www.baeldung.com/hibernate-date-time

Reply:
I'll check about applying Timestamp type on birthday.